ReadTestFile.java

```diff
@@ -0,0 +1,41 @@
+package tasks;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.BufferedReader;
+
+/**
+ * This program reads 'myTestFile.txt' file and prints it line by line.
+ *  
+ * @author Jayesh Soni
+ * @since 2024-04-23
+ */
+public class ReadTestFile {
+
+	public static void main(String[] args)  throws IOException{
+		FileReader fileReader = null;
+		BufferedReader bufferedReader = null;
+
+		try {
+			// Create FileReader instance
+			fileReader = new FileReader("myTestFile.txt");
```

Comment:
You can give comment here after complete the try like like this 

fileReader = new FileReader("myTestFile.txt"); // Create FileReader instance

Apply this in the all the practical which you have used try catch and finally. Applyy in this three block.